test/DynamoCoreTests/SerializationTests.cs

```diff
@@ -631,6 +631,54 @@ public void NodeDescriptionDeserilizationTest()
             Assert.AreEqual(node.Description, "Makes a new list out of the given inputs");
         }
 
+        [Test]
+        public void NodeFreezeStateDeserilizationTest()
+        {
+            // The freeze state of the node is saved in node view block. However the property on node model
+            // will impact headless Dynamo clients graph run, e.g. DynamoPlayer, Refinery.
+            // This test will make sure that the isFrozenExplicitly is read from the node view block in Json file.
+            var testFile = Path.Combine(TestDirectory, @"core\serialization\NodeDescriptionDeserilizationTest.dyn");
+            OpenModel(testFile);
+            var node = this.CurrentDynamoModel.CurrentWorkspace.Nodes.First();
+            Assert.AreEqual(node.isFrozenExplicitly, true);
+        }
+
+        [Test]
+        public void NodeIsSetAsInputStateDeserilizationTest()
+        {
+            // The IsSetAsInput state of the node is saved in node view block. However the property on node model
+            // will impact headless Dynamo clients graph run, e.g. DynamoPlayer, Refinery.
+            // This test will make sure that the IsSetAsInput is read from the node view block in Json file.
+            var testFile = Path.Combine(TestDirectory, @"core\serialization\NodeDescriptionDeserilizationTest.dyn");
+            OpenModel(testFile);
+            var node = this.CurrentDynamoModel.CurrentWorkspace.Nodes.First();
+            Assert.AreEqual(node.IsSetAsInput, true);
```

Comment:
This test is failing here for some reason I cant believe at all :(

Reply:
After deleting the Dynamo build and re run the test, still fails.. I have checked everywhere in code where the setter might be called but did not find anything anywhere...

Reply:
@mjkkirschner Yup, List Create can't be set as Input even I did such in the Json. I added a number node and that works